source

```diff
@@ -93489,9 +93489,10 @@ location.href = '#foo';</code></pre>
        <var>navTimingType</var>. The inline content should indicate to the user the sort of error
        that occurred.</p></li>
 
-       <li><p>Set <var>entry</var>'s <span data-x="she-document-state">document state</span>'s
-       <span data-x="document-state-document">document</span>'s <i
-       data-x="concept-document-salvageable">salvageable</i> to false.</p></li>
+       <li><p><span>Make document unsalvageable</span> given <var>entry</var>'s <span
+        data-x="she-document-state">document state</span>'s <span
+        data-x="document-state-document">document</span> and a string "<code
+        data-x="blocking-error-document">error-document</code>".</p></li>
```

Comment:
Maybe make the string "the navigation failed" or something instead of "the document had an error" (since we're showing an error page because the navigation failed?)